pkg/kubelet/types/types.go

```diff
@@ -58,7 +58,7 @@ func (t *Timestamp) Get() time.Time {
 // GetString returns the time in the string format using the RFC3339NanoFixed
 // layout.
 func (t *Timestamp) GetString() string {
-	return t.time.Format(RFC3339NanoFixed)
+	return t.time.UTC().Format(RFC3339NanoFixed)
```

Comment:
how is this method used? what are the implications of changing from local to utc here?

Reply:
It is recorded in the annotation of the pod

https://github.com/kubernetes/kubernetes/blob/2f753ec4c826895e4ccd3d6bdda2b1ab777ceeb8/pkg/kubelet/config/config.go#L384

The annotation is used here, parsed by `time.Parse`. So it looks safe
https://github.com/kubernetes/kubernetes/blob/49897ca156c88a3c7a21681bde0b895041db682f/pkg/kubelet/kubelet.go#L1534

Reply:
This type is used in more places and has more entry points than just this string method. I don't think we should change this as part of this PR. If you want to pursue changing this, split it into its own PR, also address the NewTimestamp and ConvertToTimestamp entry points, resolve all the callers of those entry points to ensure that is safe, and get node/kubelet reviewers to approve

Reply:
> This type is used in more places and has more entry points than just this string method. I don't think we should change this as part of this PR. If you want to pursue changing this, split it into its own PR, also address the NewTimestamp and ConvertToTimestamp entry points, resolve all the callers of those entry points to ensure that is safe, and get node/kubelet reviewers to approve

Thank you very much, I understand